tabpy-tools/tabpy_tools/rest.py

```diff
@@ -1,21 +1,25 @@
 import logging as _logging
 from future.utils import with_metaclass
 import requests
+from re import compile as _compile
```

Comment:
Same comment on private namespacing, though probably best just to import `re`

Reply:
fixed